files/justfiles/toggles.just

```diff
@@ -222,14 +283,13 @@ toggle-bash-environment-lockdown:
 
     # User specific environment and startup programs
         " > ~/.bash_profile
-
         for file in "${BASH_ENV_FILES[@]}"; do
-            run0 chattr +i "$file"
+            chattr +i "$file"
```

Comment:
GitHub doesn't allow one to unresolve resolved comments.

If a system is in an inconsistent state (some files locked and some unlocked), an attempt to lock an already immutable file will result in a failure. Locking must preventively unlock to avoid such a situation.

Repeated `chattr +i` doesn't fail. Output into an immutable file `"..." > ~/.bash_profile` a couple of lines above it does.